### PR TITLE
Update protocol.md

### PR DESCRIPTION
**Pull request by jleebee**
English corrections

### CODE REVIEW

File: docs/developers/protocol/hermez-protocol/protocol.md [hunk 1]
```diff
@@ -217,15 +217,15 @@ The exit tree has the same leaf structure as the state tree with some particular
...
 
 ## Account types
 
 ### Regular rollup account
 
-Regular accounts contain an ethereum address and a babyjubjub public key.  Accounts are always indexed by ethereum address in the UX, so it is a requirement that the ethereum address authorizes the account keys.  Once the account is created, the ethereum key is used to authorize L1 txs and the babyjubjub key is used to authorize L2 txs.
-There are two ways to authorize an account creation (that is, an ethereum address authorizes the creation of an account containing that same ethereum address and a babyjubjub public key):
-- Via ethereum transaction, which has an implicit signature of the ethereum address.  This requires the owner of the ethereum address to sign the smart contract transaction call
+Regular accounts contain an Ethereum address and a babyjubjub public key.  Accounts are always indexed by Ethereum address in the UX, so it is a requirement that the ethereum address authorizes the account keys.  Once the account is created, the ethereum key is used to authorize L1 txs and the babyjubjub key is used to authorize L2 txs.
```
Comment (druiz0992) [inline docs/developers/protocol/hermez-protocol/protocol.md:226]: ethereum address authorizes => Ethereum address authorizes

the ethereum key is used to authoriz => the Ethereum key is used to....

File: docs/developers/protocol/hermez-protocol/protocol.md [hunk 2]
```diff
@@ -217,15 +217,15 @@ The exit tree has the same leaf structure as the state tree with some particular
...
+There are two ways to authorize an account creation (that is, an Ethereum address authorizes the creation of an account containing that same ethereum address and a babyjubjub public key):
```
Comment (druiz0992) [inline docs/developers/protocol/hermez-protocol/protocol.md:227]: containing that same ethereum => containing that same Ethereum...

Comment (druiz0992) [inline docs/developers/protocol/hermez-protocol/protocol.md:227]: Regarding babyjubjub. Is there an agreement to write it lowercase? i have seen it written capitalized some times

File: docs/developers/protocol/hermez-protocol/protocol.md [hunk 3]
```diff
@@ -526,7 +526,7 @@ Account could be created for a given:
 ### L2
 All L2 transactions are sent to the coordinators by the users. The coordinator collects them into a batch in order to forge it. 
 The coordinator must check that it collects valid transactions that must not perform an invalid transition state. Otherwise, the proof computed by the coordinator will not be valid.
-The user could submit any transaction data to the coordinator but it will be rejected if the transaction could not be processed. Therefore, it is on the user benefit to provide valid transaction if they want it to be inserted in the zkRollup.
+The user could submit any transaction data to the coordinator but it will be rejected if the transaction could not be processed. Therefore, it is in the users benefit to provide a valid transaction if they want it to be inserted in the zkRollup.
```
Comment (druiz0992) [inline docs/developers/protocol/hermez-protocol/protocol.md]: users' benefit?

File: docs/developers/protocol/hermez-protocol/protocol.md [hunk 4]
```diff
@@ -720,7 +720,7 @@ When a user calls a function that adds an `L1UserTx`, the following happens:
     - `position`
 
 ### L1 coordinator transactions
-Coordinator could perform some special transactions to trigger L1 transactions. This transactions are processed in the `forgeBatch` smart contract method, and all the neccesary data is provided in the method inputs. This means that like L2 transactions, the data availability can be retieved by inspecting the ethereum transaction.
+Coordinator could perform some special transactions to trigger L1 transactions. This transactions are processed in the `forgeBatch` smart contract method, and all the neccesary data is provided in the method inputs. This means that like L2 transactions, the data availability can be retieved by inspecting the Ethereum transaction.
```
Comment (druiz0992) [inline docs/developers/protocol/hermez-protocol/protocol.md]: `These` transactions are processed...

**Review by druiz0992 — CHANGES_REQUESTED** [review]
some minor typos.

File: docs/developers/protocol/hermez-protocol/protocol.md (hunk 2 above)
Comment (jleebee) [reply docs/developers/protocol/hermez-protocol/protocol.md:227]: Marta shared that the official name is  "Baby Jubjub" so perhaps we should go with that?

File: docs/developers/protocol/hermez-protocol/protocol.md (hunk 3 above)
Comment (jleebee) [reply docs/developers/protocol/hermez-protocol/protocol.md]: Yes, that is correct. I missed that one, thanks!

File: docs/developers/protocol/hermez-protocol/protocol.md (hunk 4 above)
Comment (jleebee) [reply docs/developers/protocol/hermez-protocol/protocol.md]: Yes, that is correct

**Review by jleebee — COMMENTED** [review]
I committed the changes noted by David

**Review by druiz0992 — APPROVED** [review]

**Review by arnaucube — APPROVED** [review]

**Pull request merged**